go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/dexidp/dex/api/v2 v2.0.0
 	github.com/felixge/httpsnoop v1.0.1
 	github.com/ghodss/yaml v1.0.0
+	github.com/go-ldap/ldap/v3 v3.2.4 // indirect
```

Comment:
Can you please run `go mod tidy`? It should make this a direct dependency and remove`gopkg.in/ldap.v2 v2.5.1`.

Reply:
done